_drafts/implementing-state-machines-in-postgresql.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title: "Implementing State Machines in PostgreSQL"
+---
+
+[Finite-state machines](https://en.wikipedia.org/wiki/Finite-state_machine)
```

Comment:
I'd use singular here: "Finite-state machine _is_ a wonderful [...] that _has_" Currently it's "they [plural] are a [singular]". Yeah, possible, but I think the singular-singular version is smoother (and wikipedia uses it) :)